hoomd/__init__.py

```diff
@@ -8,8 +8,23 @@
 simulations using HOOMD.
 """
 import sys
+import pathlib
 import os
 
+if ((pathlib.Path(__file__).parent / 'CMakeLists.txt').exists()
+        and 'SPHINX' not in os.environ):
+    print("It appears that hoomd is being imported from the source directory:")
+    print(pathlib.Path(__file__).parent)
+    print()
+    print("""Compile the package and import from the build directory or install
+the package and import from the Python environment.
+
+To run pytest, compile and execute `python3 -m pytest <build-directory>/hoomd`
+or install and execute, ensure your current working directory is outside the
+hoomd source directory and run `python3 -m pytest --pyargs hoomd`.
+""",
```

Comment:
Having trouble parsing the second option in this sentence. After trying some different ways, does this capture it?
```suggestion
To run pytest, either:
(1) compile then execute `python3 -m pytest <build-directory>/hoomd` or
(2) compile and install. Then, ensuring your current working directory is outside the
hoomd source directory, execute `python3 -m pytest --pyargs hoomd`.
""",
```